internal/pkg/cli/job_delete.go

```diff
@@ -253,18 +268,25 @@ func (o *deleteJobOpts) deleteJobs(envs []*config.Environment) error {
 			return err
 		}
 		// Delete job stack
-		o.spinner.Start(fmt.Sprintf(fmtJobDeleteStart, o.name, env.Name))
+		o.spinner.Start(fmt.Sprintf(fmtJobStackDeleteStart, o.name, env.Name))
 		if err = o.deleteStack(sess, env.Name); err != nil {
-			o.spinner.Stop(log.Serrorf(fmtJobDeleteFailed, o.name, env.Name, err))
+			o.spinner.Stop(log.Serrorf(fmtJobStackDeleteFailed, o.name, env.Name, err))
 			return err
 		}
-		o.spinner.Stop(log.Ssuccessf(fmtJobDeleteComplete, o.name, env.Name))
+		o.spinner.Stop(log.Ssuccessf(fmtJobStackDeleteComplete, o.name, env.Name))
+		// Delete orphan tasks
+		o.spinner.Start(fmt.Sprintf(fmtJobTasksDeleteStart, o.name, env.Name))
```

Comment:
Is there some way to only display the text of `fmtJobTasksDeleteStart` if there are actually orphan tasks to be deleted? This will print every time a customer runs `job delete` regardless of whether any task is still running and may give the impression that a running task was deleted even when that wasn't the case.

Reply:
good idea!